packages/adapter/addon/json-api.js

```diff
@@ -218,9 +218,35 @@ const JSONAPIAdapter = RESTAdapter.extend({
 
     @property coalesceFindRequests
     @type {boolean}
-  */
+    */
   coalesceFindRequests: false,
 
+  /**
+    Called by the store in order to fetch several records together if `coalesceFindRequests` is true
+    For example, if the original payload looks like:
+    ```js
+    {
+      "id": 1,
+      "title": "Rails is omakase",
+      "comments": [ 1, 2, 3 ]
+    }
+    ```
+    The IDs will be passed in this form:
+    ```
+    filter[id]=1,2,3
```

Comment:
The example here appears to have very little to do with how this method operates and even less to do with JSON:API format. I'd suggest looking at the `findMany` documentation from the `MinimumAdapterInterface` for inspiration.